ui/component/common/form-components/form-field.jsx

```diff
@@ -83,16 +96,20 @@ export class FormField extends React.PureComponent<Props> {
       label,
       labelOnLeft,
       name,
-      noEmojis,
       postfix,
       prefix,
       quickActionLabel,
       stretch,
       textAreaMaxLength,
       type,
-      openEmoteMenu,
+      slimInput,
+      commentSelectorsProps,
+      showSelectors,
       quickActionHandler,
+      setShowSelectors,
       render,
+      handleTip,
+      handleSubmit,
```

Comment:
Probably ok for now since don't have time, but components in the `common/` folder should be designed as general-purpose as possible -- i.e. it should never know what it is used for.  Seeing props like `isLivestream`, `commentSelectrosProps` and `handleTip` breaks the design.  It will eventually end up bloated like `ClaimListDiscover`.  It also makes it hard to code-split since the component bundles everything.

Specific usage should be encapsulated by a parent component, leaving FormField to do as little as possible 
e.g. CommentCreate over FormField, or FormFieldPrice extending the capabilities of FormField.

Reply:
True I wanted to do that like I did to the drawers etc. but ended up just making a wrapper component for the textarea itself